perfkitbenchmarker/benchmark_spec.py

```diff
@@ -116,6 +119,13 @@ def UnPickleLock(locked, *args):
         },
         FIREWALL: openstack_network.OpenStackFirewall
     },
+    CLOUDSTACK: {
+        VIRTUAL_MACHINE: {
+            DEBIAN: cs_vm.DebianBasedCloudStackVirtualMachine,
+            RHEL: cs_vm.CloudStackVirtualMachine
+        },
+        FIREWALL: cs_nw.CloudStackFirewall
```

Comment:
When you merge with master you'll want to delete this, but add NETWORK: cs_nw.CloudStackNetwork